broker-cli/utils/basic-auth.js

```diff
@@ -0,0 +1,48 @@
+const grpc = require('grpc')
+
+/**
+ * @constant
+ * @type {String}
+ * @default
+ */
+const BASIC_AUTH_PREFIX = 'Basic'
+
+/**
+ * @constant
+ * @type {String}
+ * @default
+ */
+const BASIC_AUTH_DELIMITER = ':'
+
+/**
+ * Creates a Basic Authentication string (Base64) from user/password credentials
+ *
+ * Example Result: `Basic
+ *
+ * @param {String} username
+ * @param {String} password
+ * @return {String} res - Basic Authentication string (RFC 7235)
+ */
+function credentialsToBasicAuth (username, password) {
+  const credentialHash = Buffer.from(`${username}${BASIC_AUTH_DELIMITER}${password}`).toString('base64')
```

Comment:
I don't like calling this a hash since it's just a base64 encoding.

Reply:
I agree. its a bad name